bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -0,0 +1,337 @@
+          </p>
+        </div>
+
+        <div class="tip">
+          <h4>{{ _('Do the two-step') }}</h4>
+          <p>
+            {{ _('For the best protection, take advantage of 2-step authentication wherever it’s offered.') }}
+            <a href="http://stopthinkconnect.org/2stepsahead/resources">{{ _('Find out more.') }}</a>
+          </p>
+        </div>
+
+        <div class="tip">
+          <h4>{{ _('In case of emergency') }}</h4>
+          <p>{{ _('Here’s what to do <a href="{url}">if you think your account has been compromised</a>.').format(url='https://www.staysafeonline.org/stay-safe-online/keep-a-clean-machine/hacked-accounts')|safe }}</p>
+        </div>
+      </div>
+
+      <div class="topic with-image">
+        <div class="topic-intro">
+          <h3>{{ _('Fix it before it breaks') }}</h3>
+          <p>
+            {{ _('Software updates are like oil changes ­– they can be a hassle in the moment, but a lifesaver in hindsight.') }}
+            {{ _('Update apps and devices regularly to take advantage of developers’ latest and greatest security enhancements.') }}
+          </p>
+        </div>
+
+        {{ high_res_img('teach/smarton/security/fix-it.png', {'width': '300', 'height': '310', 'class': 'topic-image', 'alt': ''}) }}
+
+        <div class="tip columns columns-2">
+          <div class="col">
+            <h4>{{ _('Browsers') }}</h4>
+            <ul>
+              <li><a href="https://support.mozilla.org/kb/update-firefox-latest-version" class="go">{{ _('Firefox update') }}</a></li>
+              <li><a href="{{ url('mozorg.plugincheck') }}" class="go">{{ _('Firefox plugin check') }}</a></li>
+            </ul>
+            <ul>
+              <li><a href="https://support.google.com/chrome/answer/95414" class="go">{{ _('Chrome update') }}</a></li>
+              <li><a href="https://support.google.com/chrome/answer/1181003" class="go">{{ _('Chrome plugin check') }}</a></li>
+            </ul>
+            <ul>
+              <li><a href="http://pcsupport.about.com/od/browsers/fl/internet-explorer-version.htm" class="go">{{ _('Internet Explorer update') }}</a></li>
+              <li><a href="http://windows.microsoft.com/en-us/internet-explorer/manage-add-ons" class="go">{{ _('Internet Explorer plugin check') }}</a></li>
+            </ul>
+            <ul>
+              <li><a href="https://support.apple.com/HT204416" class="go">{{ _('Safari update') }}</a></li>
+              <li><a href="https://support.apple.com/HT202819" class="go">{{ _('Safari plugin check') }}</a></li>
+            </ul>
+          </div>
+
+          <div class="col">
+            <h4>{{ _('Operating systems') }}</h4>
+            <ul>
+              <li><a href="http://windows.microsoft.com/en-US/windows7/products/features/windows-update" class="go">Windows</a></li>
+              <li><a href="https://support.apple.com/en-us/HT201541" class="go">Mac</a></li>
```

Comment:
Looks like we can omit `en-us` in this apple link (the windows one about is english only though).